plugins/Insights/javascripts/insightsDataTable.js

```diff
@@ -60,9 +60,6 @@
                 $('td.label', domElem).width(parseInt(width * 0.50, 10));
             }
 
-            var showScrollbarIfMoreThanThisPxOverlap = 32;
```

Comment:
@sgiehl is removing this maybe causing https://builds-artifacts.matomo.org/matomo-org/matomo/3.x-dev/36806/UIIntegrationTest_dashboard2.png ?

Reply:
not sure. need to investigate that

Reply:
should be fixed with the last commit